cpg-core/src/test/resources/lambda.cpp

```diff
@@ -0,0 +1,5 @@
+int main() {
```

Comment:
somehow we have two different lambda and lambdas.cpp and `testLambdas`in CXXLanguagueFrontend and some other test.